### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by sukaant**
SmartColumbusOS/scosopedia#279 co-authored-by: Matt Dunson dunson.142000@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ Finally, user access, discovery, and analysis is facilitated by a ReactJS web ap
 a RESTful API, or a web socket API for streaming data feeds.
 
 ## local development
+### local environment setup
+* [Setup needed to prepare workstation](https://github.com/Datastillery/smartcitiesdata/wiki/Setup)
+* [Windows](https://github.com/Datastillery/smartcitiesdata/wiki/Windows-Setup)
+* [macOS](https://github.com/Datastillery/smartcitiesdata/wiki/macOS-Setup)
+* [Linux](https://github.com/Datastillery/smartcitiesdata/wiki/Linux-Setup)
+
```
Comment (dunson062786) [inline README.md:36]: Can we also update the content in these links. The directions only tell you how to install the software you'll need for the platform they don't actually give you directions on how to setup the environment.

**Review by dunson062786 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dunson062786) [inline README.md:36]: Can you include a link to https://github.com/Datastillery/smartcitiesdata/wiki/Run

**Review by dunson062786 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,27 +28,30 @@ Finally, user access, discovery, and analysis is facilitated by a ReactJS web ap
...
+| estuary           | [Click Here](https://github.com/Datastillery/smartcitiesdata/blob/master/apps/estuary/README.md)           |
+| forklift          | [Click Here](https://github.com/Datastillery/smartcitiesdata/blob/master/apps/forklift/README.md)          |
+| odo               | [Click Here](https://github.com/Datastillery/smartcitiesdata/blob/master/apps/odo/README.md)               |
+| reaper            | [Click Here](https://github.com/Datastillery/smartcitiesdata/blob/master/apps/reaper/README.md)            |
+| valkyrie          | [Click Here](https://github.com/Datastillery/smartcitiesdata/blob/master/apps/valkyrie/README.md)          |
 
 ### notes
 ws://localhost:8087/socket/websocket
+
+## To run E2E Test Locally
```
Comment (JarredOlson) [inline README.md:56]: Use `End to End` instead of `E2E`

File: apps/discovery_api/README.md [hunk 3]
```diff
@@ -3,10 +3,12 @@
 Discovery API serves as middleware between our data storage and our Discovery UI.
 
 ### To run locally:
-  * `mix deps.get`
-  * `MIX_ENV=integration mix docker.start`
-  * `MIX_ENV=integration iex -S mix start --config config/auth0.exs`
-  * `MIX_ENV=integration mix docker.stop`
+  * Install dependencies with `mix deps.get` (in the smartcitiesdata directory)
+  * `MIX_ENV=integration mix docker.start` (in this directory)
+  # TODO: Create a config/auth0.exs file and test it
```
Comment (JarredOlson) [inline apps/discovery_api/README.md:8]: Update or Remove TODO comment

File: apps/discovery_api/README.md [hunk 4]
```diff
@@ -3,10 +3,12 @@
...
+  * `MIX_ENV=integration iex -S mix start --config config/auth0.exs` (in this directory)
+    or just `MIX_ENV=integration iex -S mix start` if you want to try it without an auth config
+  * `MIX_ENV=integration mix docker.stop` (in this directory)
```
Comment (JarredOlson) [inline apps/discovery_api/README.md:11]: Let's call out that the above steps start it and this step stops it

File: apps/discovery_api/README.md [hunk 5]
```diff
@@ -27,32 +29,46 @@ Discovery API serves as middleware between our data storage and our Discovery UI
   * Run `mix test.watch --stale` to only rerun the tests for modules that have changes
   * Run `mix test.integration` to run the integration tests
 
-### To run inside a container(from the root directory):
-  * `docker build . -t <image_name:tag>`
+### To run inside a container:
+  * Go up to the smartcitiesdata directory `cd ../../`
+  * `./scripts/build.sh discovery_api 1.0`
+  * TODO: update this to include actual running the container
```
Comment (JarredOlson) [inline apps/discovery_api/README.md:35]: Update or Remove TODO comment

File: apps/discovery_api/README.md [hunk 6]
```diff
@@ -27,32 +29,46 @@ Discovery API serves as middleware between our data storage and our Discovery UI
...
+    ```elixir
+      "localhost:#{Application.get_env(:telemetry_event, :metrics_port)}/metrics"
+    ```
+    and go to the url printed out it in your browser.
   * Go to http://localhost:4000/api/v2/dataset/search
   * You can get paginated results using the url http://localhost:4000/api/v2/dataset/search?offset=10&limit=5&sort=name_asc
 
 ### Tableau Web Data Connector
 This application hosts a Tableau Web Data Connector that uses this API for interfacing with Tableau. More information can be found in its [README](./priv/static/tableau/README.md)
+<!--- TODO: Ask Jarred if he wants more details in the Tableau readme or if it is fine the way it is. -->
```
Comment (JarredOlson) [inline apps/discovery_api/README.md:48]: What is there is fine for now

File: apps/reaper/README.md [hunk 7]
```diff
@@ -12,7 +12,9 @@ Retrieves data, decodes it, and loads it onto a Kafka topic
   * Run `mix test.integration` to run the integration tests
 
 ## To run inside a container(from the root directory):
-  * `docker build . -t <image_name:tag>`
+  * Go up to the smartcitiesdata directory `cd ../../`
+  * `./scripts/build.sh reaper 1.0`
+  * TODO: update this to include actual running the container
```
Comment (JarredOlson) [inline apps/reaper/README.md:17]: Update or Remove TODO

**Pull request merged**